tldr/cli.py

```
@@ -578,12 +600,11 @@ def resolve_language(lang_arg: str, project_path: str | Path) -> str:
             from .semantic import _detect_project_languages
             respect_ignore = not getattr(args, 'no_ignore', False)
             langs = _detect_project_languages(project_path, respect_ignore=respect_ignore)
-            return langs[0] if langs else "python"
-        elif lang_arg == "all":
-            from .semantic import _detect_project_languages
-            respect_ignore = not getattr(args, 'no_ignore', False)
-            langs = _detect_project_languages(project_path, respect_ignore=respect_ignore)
-            return langs[0] if langs else "python"
+            if langs:
+                return langs[0]
+            else:
```

Ruff (0.15.5)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)